cmd/activator/main.go

```diff
@@ -285,6 +285,7 @@ func main() {
 		name, server := "https", pkgnet.NewServer(":"+strconv.Itoa(networking.BackendHTTPSPort), ah)
 		go func(name string, s *http.Server) {
 			s.TLSConfig = &tls.Config{
+				//MinVersion:     tls.VersionTLS13,
```

Comment:
Thank you! Could you open an github issue for the tracker and add a comment with the issue number?
(If you can fix this soon, you don't need to open the issue, though.)